R/prepare_data.R

```diff
@@ -139,6 +148,12 @@ prepare_data.incidence2 <- function(data, cases_variable = "cases",
     data,
     fill = ifelse(fill_NA, 0, NA_integer_)
   )
+  if (fill_NA) {
+    message(
```

Comment:
I haven't given it a lot of thought, or used the function interactively so could definitely be wrong, but this feels like a `warning()` might be more appropriate, depending on how much the calculation is dependent on missing values being assumed zero.

Reply:
No calculations in this function depend on zero filling, but downstream ones expect that there are no NAs. So the only choice is to fill with zeroes or to filter out the data. I'm not sure what to do here, there are arguments for both

Reply:
I've left this as is for now, we can always upgrade this to a warning in future versions.